src/RealtimeClient.ts

```diff
@@ -119,6 +124,8 @@ export default class RealtimeClient {
    * @param options.encode The function to encode outgoing messages. Defaults to JSON: (payload, callback) => callback(JSON.stringify(payload))
    * @param options.decode The function to decode incoming messages. Defaults to Serializer's decode.
    * @param options.reconnectAfterMs he optional function that returns the millsec reconnect interval. Defaults to stepped backoff off.
+   * @param options.worker Use Web Worker to set a side flow. Defaults to false.
+   * @param options.workerUrl The URL of the worker script. Defaults to https://realtime.supabase.com/worker.js that includes a heartbeat event call to keep the connection alive.
```

Comment:
i don't really see a reason to have these options. if we enable [this](https://github.com/supabase/realtime-js/pull/419/files#r1739488520) then i don't see why devs would want to bring their own worker.

also see [this](https://github.com/supabase/realtime-js/pull/419/files#r1739484492) where we automate it for them.

this is something that devs shouldn't have to think about.

Reply:
this is more of a preventive measure so users have a way to override for testing or some other situation they need.

In theory for the default implementation they just need to say `true` and that is, just works.